package.json

```diff
@@ -48,6 +48,10 @@
     "babelify": "~7.3.0",
     "browserify": "13.0.0",
     "cartoassets": "CartoDB/CartoAssets#master",
+    "eslint-config-standard": "^10.2.1",
```

Comment:
`eslint-config-semistandard`?

Reply:
They are peer-dependencies of [eslint-confi-semistandard](https://github.com/Flet/eslint-config-semistandard#install)